template/deployments/helm/charts/values.yaml

```diff
@@ -1,16 +1,16 @@
-# Default values for {{APPNAME}}.
```

Comment:
This values.yaml is going to be part the majority of our gotemplates definition.  This means we need to have all the variables defined in here.